docs/setup.rst

```diff
@@ -0,0 +1,30 @@
+Install CellProfiler
+====================
+
+In this section, we indicate how to install CellProfiler in a `Conda <https://conda.io/en/latest/>`_ environment.
+We will use the CellProfiler API to analyze data stored in an OMERO server.
+
+CellProfiler currently runs on Python 2.7. It does not yet support Python 3.
+
+
+**Setup**
+---------
+
+We recommand to install CellProfiler using Conda.
+Conda manages programming environments in a manner similar to 
+`virtualenv <https://virtualenv.pypa.io/en/stable/>`_.
+
+- We assume that you have a local copy of the `omero-guide-cellprofiler repository <https://github.com/ome/omero-guide-cellprofiler>`_. If not, first clone the repository::
```

Comment:
`` We assume that you have a local copy of the `omero-guide-cellprofiler repository <https://github.com/ome/omero-guide-cellprofiler>`_. If not, first clone the repository::`` ->
``If you do not  have a local copy of the `omero-guide-cellprofiler repository <https://github.com/ome/omero-guide-cellprofiler>`_,  make it by cloning that repositoryL``
(simplify the sentence)